site-src/index.md

```diff
@@ -94,8 +94,32 @@ This flexibility allows the API to adapt to vastly different
 organizational models and implementations while remaining a portable and
 standard API.
 
+## What's the difference between Gateway API and an API Gateway?
 
-## Who is working on Gateway?
+The concept of a Gateway is a networking component that allows data to transfer from one
+network to another network.
+
+An API Gateway is a Gateway implementation that deals more specifically with backend services
```

Comment:
imo this doesnt quite feel right. 
I asked ChatGPT what an API Gateway was and here is its answer
<img width="669" alt="Screenshot 2023-07-20 at 10 37 58 AM" src="https://github.com/kubernetes-sigs/gateway-api/assets/9469142/f27baa4f-8d48-4a0f-bf39-374bc125bc91">

a suggestion would be to consider reframing this as API Gateway is a Gateway implementation that deals with Request Routing, Security and many more things like Caching, Monitoring, Transformations where as Gateway API is an interface focused on defining standards for networking and security settings associated with Ingress and the Mesh

Reply:
I see what you're getting at, focusing on the features.

My intention here was to distinguish between a gateway (abstract networking thing) and an API gateway (networking thing that exposes backend services via APIs).
My preference was for less detail, and let the reader do their own research on what are the detailed features an API Gateway might provide are. 

If you'd like to suggest any changes though via the github suggestion feature, that would allow others to chime in more easily too.